src/components/Shared/DecisionResult.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import LegislationAdoptedIcon from "../Cards/LegislationCard/LegislationAdoptedIcon";
+import LegislationRejectedIcon from "../Cards/LegislationCard/LegislationRejectedIcon";
+import LegislationInProgressIcon from "../Cards/LegislationCard/LegislationInProgressIcon";
+import { MATTER_STATUS_DECISION } from "../../constants/ProjectConstants";
+import { VOTE_DECISION } from "../../constants/ProjectConstants";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+interface DecisionResultProps {
+  /**Result of the vote or council decision.  This is an enum value, you can see the enumeration in ProjectConstants */
+  result: string;
+}
+
+const DecisionResult = ({ result }: DecisionResultProps) => {
+  let statusIcon = <div />;
+  switch (result) {
+    case VOTE_DECISION.APPROVE:
+      statusIcon = <LegislationAdoptedIcon />;
+      break;
+    case VOTE_DECISION.REJECT:
+      statusIcon = <LegislationRejectedIcon />;
+      break;
+    case MATTER_STATUS_DECISION.IN_PROGRESS:
+      statusIcon = <LegislationInProgressIcon />;
+      break;
+    default:
+      break;
```

Comment:
I don't think the case logic here is quite complete.

For example, for the first case, the result can be VOTE_DECISION.APPROVE or MATTER_STATUS_DECISION.ADOPTED?

Reply:
I added the missing cases.